std/typecons.d

```diff
@@ -1859,7 +1859,7 @@ unittest
   Returns:
       A string to be mixed in to an aggregate, such as a `struct` or `class`.
 */
-string alignForSize(E...)(string[] names...)
+string alignForSize(E...)(const char[][] names...)
```

Comment:
@atilaneves `const(string)[]` is equivalent to `const(immutable(char)[])[]`, it still would not work because  you cannot convert `const(char)[]` to `immutable(char)[]`. You need a `const` view of the characters themselves, not of the slice that refers to them; hence `const(char)[]`.